src/plugins/StateService.js

```diff
@@ -11,7 +12,17 @@ const StateService = {
     Vue.prototype.$push = function () {
       const nextStep = next(this.$route, this.$store.getters.getAllSteps)
       this.$router.push(nextStep.path).catch((failure) => {
-        throw new Error(failure)
+        if (
+          VueRouter.isNavigationFailure(
+            failure,
+            VueRouter.NavigationFailureType.cancelled
+          )
+        ) {
+          this.$matomo &&
+            this.$matomo.trackEvent("Parcours", "Navigation cancelled", failure)
```

Comment:
À quoi ressemble un évènement comme ça enregistré dans Matomo ? Le passage de `failure` génère quoi ?

Reply:
j'ai changé pour failure.toString() voila l'exemple de resultat : 
`"Error: Navigation cancelled from \"/\" to \"/simulation/individu/demandeur/date_naissance\" with a new navigation."`

Reply:
Lorsqu'on fail la navigation de la home vers date_naissance

Reply:
Merci, ça permettra peut-être de voir s'il y a des pages particulièrement problématiques.